includes/modules/Charts/style.css

```diff
@@ -1,3 +1,45 @@
+/* Font family -------------------------------------------*/
+h1 {
```

Comment:
Hey Leon, we cannot add this style rules because they are overwriting the theme of the embedding site.
Further, the code can be reduced a lot by aggregating the common rules.
```
h1,h2,h3,p,blockquote,pre {
  font-family: 'Open Sans';
  font-style: normal;
  font-variant: normal;
}
h1 {
  font-size: 24px;
  font-weight: 700;
  line-height: 26.4px;
}
h2 ...etc
```
Last but not least we should avoid using `px` because then you need special rules for mobile. If we use `em` or `rem` the font size scale relatively to the parent font or global font size.